### PR TITLE
JOOQ @Generated annotation

### PR DESCRIPTION
**Pull request by robfletcher**
Now we're running on Java 11 we can have JOOQ add the @Generated annotation to generated source files

### CODE REVIEW

File: keel-clouddriver/keel-clouddriver.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,6 @@ dependencies {
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
   implementation("com.github.ben-manes.caffeine:caffeine")
   implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8")
```
Comment (robfletcher) [inline keel-clouddriver/keel-clouddriver.gradle.kts:15]: `kotlinx-coroutines-jdk8` transitively pulls in `kotlinx-coroutines-core` so there's no need to specify both

File: keel-sql/src/main/resources/jooqConfig.xml [hunk 2]
```diff
@@ -30,9 +30,8 @@
     <generate>
       <javaTimeTypes>true</javaTimeTypes>
 
-      <!-- this can be reenabled once cross-compilation to Java 8 is no longer required -->
-      <!-- generatedAnnotation>true</generatedAnnotation>
-      <generatedAnnotationType>DETECT_FROM_JDK</generatedAnnotationType -->
+      <generatedAnnotation>true</generatedAnnotation>
+      <generatedAnnotationType>DETECT_FROM_JDK</generatedAnnotationType>
```
Comment (robfletcher) [inline keel-sql/src/main/resources/jooqConfig.xml:34]: Cross compilation is no longer required, so…

**Review by plumpy — APPROVED** [review]
Whoops, sorry I forgot to undo that.

**Comment by robfletcher** [comment]
> Whoops, sorry I forgot to undo that.

No worries, I don't think it's a big deal. Just happened to notice it when I was having a weird transient issue with the `jooqGenerate` task.

**Pull request merged**